lib/jormungandr/test/integration/Cardano/Faucet.hs

```diff
@@ -0,0 +1,180 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Cardano.Faucet
+    ( initFaucet
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.Network
+    ( NetworkLayer )
+import Cardano.Wallet.Primitive.Mnemonic
+    ( mkMnemonic )
+import Control.Concurrent.MVar
+    ( newMVar )
+import Test.Integration.Faucet
+    ( Faucet (..) )
+
+
+-- | Initialize a bunch of faucet wallets and make them available for the
+-- integration tests scenarios.
+initFaucet :: NetworkLayer t IO -> IO Faucet
+initFaucet _nl = do
```

Comment:
What's this `NetworkLayer` argument here for :thinking: ?

Reply:
no needed at this moment, hence I removed it